.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,14 @@
+name: Lighthouse CI
+on: push
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Audit URLs using Lighthouse
+        uses: treosh/lighthouse-ci-action@v11
+        with:
+          urls: "https://ethereum.org/en/"
```

Comment:
Can we also add a page from the docs? since that is the most used template. Any improve we do in there will impact in a lot of pages.
E.g. https://ethereum.org/en/developers/docs/intro-to-ethereum/

Reply:
Oh yeah... I missed this, but I thought the point of this tool was to be able to run it on more pages than the homepage, no?

To Pablo's point, it'd be great to see a representative sampling of pages... ie, at least one from each template type and at least one RTL language